x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -52,6 +55,56 @@ const WatchListUi = ({ intl }: { intl: InjectedIntl }) => {
     [watches, deletedWatches]
   );
 
+  const createWatchButtons = (
+    <EuiFlexGroup gutterSize="m">
```

Comment:
We need to apply `justifyContent="center"` to this flex group when the buttons are presented in the empty state in order to align them along the vertical axis with the other content:

![image](https://user-images.githubusercontent.com/1238659/58223769-dda28180-7ccf-11e9-862f-ffeebd1c0b21.png)

I'm not sure if this prop will make sense when the buttons are used elsewhere, so maybe we'll have to wrap them in a function that lets you conditionally apply this prop.

Reply:
good catch! i didn't see any adverse affects with having `justifyContent="center"` applied when the buttons are aligned with the search so i did not add any conditionally logic.